mappings/net/minecraft/util/ItemActionResult.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/unmapped/C_gqmsalid net/minecraft/util/ItemActionResult
```

Comment:
I like `interaction` better than `action` here, although I agree with clarifying it